### ISSUE TITLE
Heroku Updates

### ISSUE THREAD

**Issue by boltron88** [AHAAAAAAA/PokemonGo-Map#2520]
Is it possible for Heroku to automatically update the script once it is deployed?

**Comment by brianjking** [comment]
I haven't found proper update documentation written if you deploy using the one click Heroku deploy button. However, updating a Heroku app once it's installed if you manually deploy it is generally very easy.

My only confusion here is with if I should be deploying from `develop` or `master` branch or if it's suggested to just use release tags (ex: `v2.1.0`)

Based on https://www.jonaharagon.com/PoGoMapWiki/#!heroku.md it looks like it's suggested to `git push heroku develop:master`, but I'm not sure if this is outdated or not?

Thanks!

**Comment by wwwescape** [comment]
i would love to know how to update when deployed via the Heroku deploy button too. Right now I just rename my old app and deploy all over again.

**Comment by leonelgalan** [comment]
It's not outdated. If that's a general _git_ command for pushing your local develop to Heroku's master, as long as you have a remote called "heroku". If you don't you can add one by running:

Once

```sh
git clone git@github.com:AHAAAAAAA/PokemonGo-Map.git
cd PokemonGo-Map/
git checkout develop
git remote add heroku git@heroku.com:your-app-name.git 
```
Every time you want to update:

```sh
git pull
git push heroku develop:master
```

**Comment by brianjking** [comment]
@leonelgalan `develop` is the correct branch to be using for my `git pull` before I redeploy to Heroku, correct?

It looks like the `config.ini` file is not of use since all of these variables seem to be set in the `environment variables` inside of the `heroku cli` or `heroku.com` web UI, correct?

Thanks!

**Comment by leonelgalan** [comment]
Yeah, all correct. I mean you can deploy production too, but usually we want the latest (and buggiest of all, is part of the fun)
